azure-spring-boot/src/test/java/com/microsoft/azure/spring/autoconfigure/aad/Constants.java

```diff
@@ -9,9 +9,12 @@
 import java.util.List;
 
 public class Constants {
+    public static final String SERVICE_INSTANCE_PROPERTY = "azure.activedirectory.instance";
```

Comment:
azure.activedirectory.region?

Reply:
I'm not quite sure about the naming, because on the [azure china intro page]
 it used `instance` (https://docs.microsoft.com/en-us/azure/china/china-welcome) _Microsoft Azure operated by 21Vianet (Azure China 21Vianet) is a physically separated `instance` of cloud services located in mainland China_, for the [regions](https://azure.microsoft.com/en-us/regions/),  it means more. How about `provider`?

Reply:
@yungez property name has been changed to `azure.activedirectory.environment`.